env/env_test.go

```diff
@@ -39,6 +39,7 @@ func (s *MySuite) TestLoadDefaultEnv(c *C) {
 
 	c.Assert(e, Equals, nil)
 	c.Assert(os.Getenv("gauge_reports_dir"), Equals, "reports")
+	c.Assert(os.Getenv("gauge_environment"), Equals, "default")
```

Comment:
why not using `common.DefaultEnvDir`?

Reply:
It's because the test specifically loads the "default" environment. It does not make sense to use the property from common as that can change.

Reply:
the test name is `TestLoadDefaultEnv`, then it should use `ommon.DefaultEnvDir` constant.

Reply:
Done.